packages/teleterm/src/ui/utils/retryWithRelogin.ts

```diff
@@ -1,6 +1,8 @@
-import {} from 'teleterm/ui/services/workspacesService/documentsService';
 import { routing } from 'teleterm/ui/uri';
 import AppContext from 'teleterm/ui/appContext';
+import Logger from 'teleterm/ui/logger';
```

Comment:
Please use `Logger` from `teleport/logger` as I'm going to remove the one you used https://github.com/gravitational/webapps/pull/853/commits/fa37504c33bd338aee213fe2b1403ec6156569dd

Reply:
You mean I should use `teleterm/logger` I assume?

Reply:
Ah, yes